grails-app/conf/UrlMappings.groovy

```diff
@@ -213,6 +213,23 @@ class UrlMappings {
             action = [DELETE: "removingItem"]
         }
 
+        // Combined shipments
+
+        "/api/orderNumberOptions"(parseRequest: true) {
```

Comment:
I'm going to review / refactor the APIs a little later. We should discuss these in an upcoming tech huddle.